pep/app/pods/application/controller.ts

```diff
@@ -39,9 +38,8 @@ export default class Application extends Controller {
     @tracked viewedCount: string = '';
     @tracked viewedPeriod: ViewPeriod = ViewPeriod.PAST_WEEK;
     @tracked searchTerms: SearchTermValue[] = [];
-
-    @tracked rightSidebarWidgets = this.configuration.base.global.cards.right;
-    @tracked leftSidebarWidgets = this.configuration.base.global.cards.left;
+    @tracked readonly rightSidebarWidgets = this.configuration.base.global.cards.right;
+    @tracked readonly leftSidebarWidgets = this.configuration.base.global.cards.left;
```

Comment:
I still think these need to be getters (or use a new `@alias` decorator that has the same behavior as the getter) because of the concerns we talked about (redundant `@tracked`, not consistent w/the existing pattern). But mainly because of the larger issue that these won't update if `this.configuration.base` updates later.